host/CMakeLists.txt

```diff
@@ -351,6 +351,7 @@ target_compile_definitions(oehost
   # package.
   $<BUILD_INTERFACE:OE_API_VERSION=2>
   PRIVATE
+  OE_CONTEXT_SWITCHLESS_EXPERIMENTAL_FEATURE
```

Comment:
Did we decide to make this preview and if so should we remove this?

Reply:
Removed.